Documentation/Helm-Charts/operator-chart.md

```diff
@@ -110,6 +110,8 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.rbdPluginUpdateStrategy` | CSI RBD plugin daemonset update strategy, supported values are OnDelete and RollingUpdate | `RollingUpdate` |
 | `csi.rbdPluginUpdateStrategyMaxUnavailable` | A maxUnavailable parameter of CSI RBD plugin daemonset update strategy. | `1` |
 | `csi.rbdPodLabels` | Labels to add to the CSI RBD Deployments and DaemonSets Pods | `nil` |
+| `csi.readAffinity.crushLocationLabels` | Define which node labels to use as CRUSH location. This corresponds to the values used in ceph CRUSH map. | labels listed [here](../CRDs/Cluster/ceph-cluster-crd.md#osd-topology) |
+| `csi.readAffinity.enabled` | Enable read affinity for rbd volumes. Recommended to set to true if running on a kernel 5.8 or newer. | `false` |
```

Comment:
```suggestion
| `csi.readAffinity.crushLocationLabels` | Define which node labels to use as CRUSH location. This should correspond to the values set in Ceph CRUSH map. | Default labels are listed [here](../CRDs/Cluster/ceph-cluster-crd.md#osd-topology) |
| `csi.readAffinity.enabled` | Enable read affinity for RBD volumes. Recommended to set to true if running on a kernel 5.8 or newer. | `false` |
```

Reply:
The following appears in the default value column, therefore it appropriately indicates the labels used as default.
"labels listed [here](../CRDs/Cluster/ceph-cluster-crd.md#osd-topology)"

incorporated other suggestions.